cityofaustin/atd-moped
# Create subcomponents for protection types

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/13225

## Testing
**URL to test:** Local

**Steps to test:**
1. Start the Hasura cluster and editor
2. Inspect the Moped data dictionary (`https://localhost:3000/moped/dev/lookups`) to see all of the new **Protection Type** subcomponent options associated with any component of these types:

- Bike Lane
- Crossing Island
- Crosswalk
- Intersection
- Pedestrian
- Sidewalk
- Trail
- Transit

3. Yes, that's a lot of subcomponents! I'm hoping we can pare this down a bit once Nathan W. is back from vacay, but this comprehensively covers everything in the Access DB.

---
#### Ship list
